examples/deepseek_on_gaudi/single_16k_len.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
+
+# set -x
+
+BASH_DIR=$(dirname "${BASH_SOURCE[0]}")
+source "$BASH_DIR"/utils.sh
+
+ray stop --force
+
+
+# DO NOT change unless you fully understand its purpose
+export HABANA_VISIBLE_DEVICES="ALL"
+export PT_HPU_ENABLE_LAZY_COLLECTIVES="true"
+export VLLM_RAY_DISABLE_LOG_TO_DRIVER="1"
+export RAY_IGNORE_UNHANDLED_ERRORS="1"
+export PT_HPU_WEIGHT_SHARING=0
+export HABANA_VISIBLE_MODULES="0,1,2,3,4,5,6,7"
+export PT_HPUGRAPH_DISABLE_TENSOR_CACHE=1
+export PT_HPU_LAZY_MODE=1
+export VLLM_MLA_DISABLE_REQUANTIZATION=1
+
+export VLLM_MOE_N_SLICE=8
+export VLLM_EP_SIZE=8
```

Comment:
I used a different name for 1.21 release, please use "VLLM_ENABLE_EXPERT_PARALLEL=1"

Reply:
default is "VLLM_ENABLE_EXPERT_PARALLEL=1"